Dockerfile

```diff
@@ -1,21 +1,27 @@
 FROM node:16-alpine
 
-WORKDIR /apps/server
+# Set environment variables
+# Environment Variable to signal that we are running production
+ENV NODE_ENV=production
+# Ontime Data path
+ENV ONTIME_DATA=/external/
+
+WORKDIR /Resources/extraResources/
 
 # Prepare UI
-COPY /apps/client/build ../client/build
+COPY /apps/client/build ./client/
 
 # Prepare Backend
-COPY /apps/server ./
+COPY /apps/server/dist/ ./server/
+COPY /demo-db/ ./preloaded-db/
+COPY /apps/server/src/external/ ./external/
 
-# Export default ports Main - OSC IN
-EXPOSE 4001/tcp 8888/udp
-ENV NODE_ENV=production
-ENV ONTIME_DATA=/server/
+# Export default ports
+EXPOSE 4001/tcp 8888/udp 9999/udp
 
-CMD ["npm", "start:headless"]
+CMD ["node", "server/docker.cjs"]
 
 # Build and run commands
 # !!! Note that this command needs pre-build versions of the UI and server apps
 # docker build -t getontime/ontime .
-# docker run -p 4001:4001 -p 10.0.0.12:8888:8888/udp --mount type=bind,source="$(pwd)/ontime-db",target=/server/preloaded-db getontime/ontime
+# docker run -p 4001:4001 -p 8888:8888/udp -p 9999:9999/udp -v ./ontime-db:/external/preloaded-db/ -v ./ontime-styles:/external/external/styles/ getontime/ontime
```

Comment:
hmm, I am unsure I understand the reason for the external directory that would make `/external/external/styles`, am I missing something?

At least in terms of semantic, the preloaded db is not external. External attempts referencing to files that the user might want to add to the distribution (hinting forwards to being able to make your own views and add here)

Perhaps the folders can be at the root like
```
app.js
docker.js
/external-styles/override.css
/db/db.json
```

Reply:
i will try to propose a more sensible path solution when we figured out the rest of your mentions